src/main/docker/jenkins/security.groovy

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+import jenkins.model.*
+import hudson.security.*
+import jenkins.security.s2m.AdminWhitelistRule
+
+def instance = Jenkins.getInstance()
+
+def hudsonRealm = new HudsonPrivateSecurityRealm(false)
+hudsonRealm.createAccount("admin", "admin")
+instance.setSecurityRealm(hudsonRealm)
+
+def strategy = new FullControlOnceLoggedInAuthorizationStrategy()
+instance.setAuthorizationStrategy(strategy)
+instance.save()
+
+Jenkins.instance.getInjector().getInstance(AdminWhitelistRule.class).setMasterKillSwitch(false)
```

Comment:
Could you document why do we turn off the master kill switch?

Reply:
No idea. This was taken from Easy2Use but just googling on it I can see that the file in E2U is just a "standard" file that can be found. I'll try removing this line and see if the tests pass.

Reply:
This line made no difference to us in the tests so I'll remove it.